app/util/math.ts

```diff
@@ -29,6 +29,10 @@ export function splitDecimal(value: number): [string, string] {
 }
 
 export function round(num: number, digits: number) {
-  const nf = Intl.NumberFormat(navigator.language, { maximumFractionDigits: digits })
+  const nf = Intl.NumberFormat(navigator.language, {
+    maximumFractionDigits: digits,
+    // very important, otherwise turning back into number will fail on > 1000 due to commas
+    useGrouping: false,
```

Comment:
Does this need to be returned as a number? I say because there's a few cases where having the grouping is preferred. `1,234 KiB` is easier to read than `1234 KiB`.

The one place we're using it could be a string: https://github.com/oxidecomputer/console/blob/25fb506d0527f5508dc90b118672c0bf8ce14dd3/app/pages/system/UtilizationPage.tsx#L250

Reply:
That is a very good point. Rounding is a display thing.